scripts/fabtests/ccm_runfabtests.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+#
+# Copyright (c) 2016 Cray Inc.  All rights reserved.
+# Copyright (c) 2016 Los Alamos National Security, LLC. All rights reserved.
```

Comment:
No need for this LANL copyright (they can add it if/when they touch this script).